bin/update_pods.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
May want to use:
```
#!/usr/bin/env python3
```

I think this would allow it to work with both (1) virtualenvs _and_ (2) Apple's built-in Python3 at `/usr/bin/python3` if the user isn't using one but at least has either Command-Line Tools or Xcode installed (which I think is a safe assumption if they're already using PodToBUILD).

If the user isn't using a virtualenv, it's safe to assume that on macOS there is a `python` on the default PATH, since neither Apple nor brew provides one (only `python3`).

Reply:
Oh nice, good to know. I'll update this, should this be done to other usages of this shebang then? I'm seeing a few others in the code.

Reply:
Also it looks like when I wrote:
> it's safe to assume that on macOS there is a python on the default PATH

I meant "it's not safe" :) It was safe up until last month, but we knew for a while it was going away.

Without being familiar at all with this repo, I'd say as a rule of thumb if a python script has been verified to be fully python3-compatible, then it should be safe to bump its shebang up to `#!/usr/bin/env python3`.

Reply:
Updated this and the rest of the scripts to `#!/usr/bin/env python3`, using the `#!/usr/bin/env python` shebang fails on macOS 12.3.